DFC.App.ContactUs/Models/ServiceOpenDetailModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
```

Comment:
Do the values of this class need to be in settings in case they change? Or potentially in Orchard Core so they can be managed in one place from different apps

Reply:
They will be changing - they will be gotten from the CMS API when it is available and not from app settings